x-pack/plugin/src/test/resources/rest-api-spec/api/async_search.submit.json

```diff
@@ -41,6 +45,15 @@
         "description":"The number of shard results that should be reduced at once on the coordinating node. This value should be used as the granularity at which progress results will be made available.",
         "default":5
       },
+      "pre_filter_shard_size":{
```

Comment:
@jimczi I was wondering if you left this out on purpose. It is settable hence I added it, let me know what you think

Reply:
We should probably prevent users to set this and assume that the can_match phase always runs on async_search. That means that we should throw an error if the value is set in the rest API but that's out of scope for this pr. +1 to remove it from the spec here though

Reply:
I will have a look at disallowing to set this parameter in a followup